### DISCUSSION TITLE
Cookie not found when doing Keycloak GUI Login

### DISCUSSION THREAD

**Discussion by AdigaAkhil** [keycloak/keycloak#22605] [Q&A]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

admin/ui

### Describe the bug

I have set up the Keycloak container behind an nginx reverse proxy, I have also assigned a separate location block for Keycloak i.e./auth/ . Consider my reverse proxy is named app.example.com.

When I go to app.example.com/auth/ keycloak's login page will open, there if I enter the username and password and if I click on login. I'll get the cookie not found error. I have searched throughout Google for this issue but have not been able to get any solution for this.
I am also forwarding all the cookies in the reverse proxy.

More details are available in this StackOverflow link,

[https://stackoverflow.com/questions/76929695/running-keycloak-container-behing-nginx-reverse-proxy-with-docker-swarm
](URL)

### Version

22

### Expected behavior

I should be able to login using user credentials into the Keycloak GUIlog in

### Actual behavior

I am getting Cookie not found because of which I am not able to login.log in

### How to Reproduce?

Running the Keycloak container behind Nginx reverse proxy will reproduce the issue. The below Stackoverflow link has all the required details

[https://stackoverflow.com/questions/76929695/running-keycloak-container-behing-nginx-reverse-proxy-with-docker-swarm](url)

**Comment by alex27riva** [comment]
Same issue, I'm running Keycloak in a Podman container.
To connect I use a hostname mapped to /etc/hosts

**Comment by codespearhead** [comment]
@AdigaAkhil You answered your own question on StackOverflow. Could you go ahead and close this discussion [2]?

> It was my mistake actually, I was ignoring the cookie headers in the nginx.conf file of the reverse proxy due to which I got the above issue. 
> proxy_ignore_headers Expires Cache-Control Set-Cookie;
> proxy_hide_header Set-Cookie;

[1] https://stackoverflow.com/a/76966732
[2] https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion
